### DISCUSSION TITLE
How to call the damage model ?

### DISCUSSION THREAD

**Discussion by nsy19971013** [peridigm/peridigm#341] [General]
I would like to use damage models other than the critical stretch model. I found other damage models in the periigm-master/src/damage folder, as shown in the following figure.
[
![image](https://github.com/peridigm/peridigm/assets/117352474/8d922cd5-f8ef-441d-ba3b-4473c4e18204)
](url)

**Comment by oldninja** [comment]
You need to make sure these are all functional and are reasonable for your model. There must be some unit tests on each one, so you can check the parameters and understand the usage.
